iphone/CoreApi/CoreApi/Formatting/DistanceFormatter.swift

```diff
@@ -0,0 +1,53 @@
+import Foundation
+
+// TODO: Replace this class with a core's implementation
```

Comment:
Did you check platform::Distance ?

Reply:
Thanks! Replaced with the core implementation.